Pracs/prac_06/car.py

```diff
@@ -0,0 +1,35 @@
+"""CP1404/CP5632 Practical - Car class example."""
+
+
+class Car:
+    """Represent a Car object."""
+
+    def __init__(self, fuel=0, name=""):
+        """Initialise a Car instance.
+
+        fuel: float, one unit of fuel drives one kilometre
+        """
+        self.fuel = fuel
+        self._odometer = 0
+        self.name = name
+
+    def add_fuel(self, amount):
```

Comment:
While this isn't required, this format may help you memorise class formatting: When using classes, def __init__ is first, then def __str__, then any other methods that may be needed. Disregard this if you'd like. :)
Otherwise, great code.